### PR TITLE
chore(setup): integrate LGTM

### PR DESCRIPTION
**Pull request by Dattaya**
@mxstbr, first it didn't work for a non default branch like you said on gitter, then I did nothing and it started to work ✨, you can see that in my experimental repository: https://github.com/Dattaya/Temp/pull/3

Also, LGTM hook appeared on branch's settings page (appeared unchecked I checked it myself):
image

#1209

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+mxstbr
+oliverturner
+justingreenberg
+gihrig
+sedubois
+chaintng
+samit4me
+amilajack
+Dattaya
```
Comment (Dattaya) [inline MAINTAINERS]: Have I missed someone?

Comment (mxstbr) [inline MAINTAINERS]: I think you got them all, nice!

**Comment by Dattaya** [comment]
Predictably, Appveyor failed due to #1246 that was fixed in dev but not in master.
:shipit:

**Comment by samit4me** [comment]
LGTM, is this something we can merge into `dev` so it will be merged to master in the next release?

**Comment by mxstbr** [comment]
I think we should merge this into `master`, otherwise it won't be enabled until the next release

**Pull request merged**